src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/integration/documents/GotenbergComponentTestBase.kt

```diff
@@ -120,14 +121,14 @@ abstract class GotenbergComponentTestBase : ComponentTestBase(useRealGotenbergSe
   protected fun uploadPartAFor(recall: RecallResponse) {
     authenticatedClient.uploadDocument(
       recall.recallId,
-      UploadDocumentRequest(DocumentCategory.PART_A_RECALL_REPORT, base64EncodedFileContents("/document/part_a.pdf"), "PART_A.pdf")
+      UploadDocumentRequest(DocumentCategory.PART_A_RECALL_REPORT, base64EncodedFileContents("/document/part_a.pdf"), FileName("PART_A.pdf"))
     )
   }
 
   protected fun uploadLicenceFor(recall: RecallResponse) {
     authenticatedClient.uploadDocument(
       recall.recallId,
-      UploadDocumentRequest(DocumentCategory.LICENCE, base64EncodedFileContents("/document/licence.pdf"), "PART_A.pdf")
+      UploadDocumentRequest(DocumentCategory.LICENCE, base64EncodedFileContents("/document/licence.pdf"), FileName("PART_A.pdf"))
```

Comment:
Tiny but this one is for a LICENSE so can we please not call it "PART_A.pdf"?  Ta.